src/components/Diagram/Diagram.js

```diff
@@ -1,33 +1,38 @@
 import React from 'react';
 import { Box, makeStyles } from '@material-ui/core';
-import * as d3 from 'd3';
-import { ReactComponent as SVGDiagram } from '../../assets/images/diagram.svg';
+//import * as d3 from 'd3';
+//import { ReactComponent as SVGDiagram } from '../../assets/images/diagram.svg';
 
-export const useStyles = makeStyles(() => ({ root: { padding: 30 } }));
+export const useStyles = makeStyles(() => ({ root: { padding: 10, height: '58vh' }, inner: {padding:10, border: 'solid'} }));
 
 const Diagram = () => {
     const classes = useStyles();
 
     React.useEffect(() => {
-        d3.select('g')
-            .selectAll('*')
-            .on('click', function() {
-                console.log(this);
-                d3.select(this)
-                    .attr('stroke', 'tomato');
-            })
-            .attr('pointer-events', 'fill')
-            .style('cursor', 'pointer');
+
     }, []);
 
     return (
         <Box
-            alignItems='center'
             className={classes.root}
             display='flex'
             flexDirection='row'
+            justifyContent = 'space-between'
         >
-            <SVGDiagram />
+            <Box
+                color = '"secondary.main"'
+                className={classes.inner}
+                alignItems='left'
+                width = {0.49}
```

Comment:
We should let layout handle the width of the boxes and not define width ourselves